### PR TITLE
Legal Textchange: New Eula (Nutzungsbedingungen) (EXPOSUREAPP-2951)

### PR DESCRIPTION
**Pull request by harambasicluka**
Updated german eula, also used the german eula to update tr & tr until we get the final version from the lawyer.

### CODE REVIEW

**Review by BMItr — APPROVED** [review]
Wonderful Eula.
 :-)

File: Corona-Warn-App/src/main/assets/terms_de.html [hunk 1]
```diff
@@ -86,127 +92,161 @@ <h2>
...
+</p>
 <p>
-    Die App berechnet auf Basis der festgestellten Begegnungen ein indikatives
-    individuelles Risiko. Hierbei werden Faktoren wie der Zeitraum seit der
-    Begegnung (<em>days since exposure</em>), die Dauer der Begegnung (<em>exposure duration</em>),
-    die ungefähre
+    Die App berechnet auf Basis der vom COVID-19-Benachrichtigungssystem
+    aufgezeichneten Begegnungen ein geschätztes individuelles Risiko. Hierbei
+    werden Faktoren wie der Zeitraum seit der Begegnung, die Dauer der
+    Begegnung, die ungefähre Nähe zur Corona-positiv getesteten Person,
```
Comment (SabineLoss) [inline Corona-Warn-App/src/main/assets/terms_de.html:239]:
```suggestion
    Begegnung, die ungefähre Nähe zur Corona-positiv getesteten Person
```

File: Corona-Warn-App/src/main/assets/terms_de.html [hunk 2]
```diff
@@ -245,61 +284,66 @@ <h1>
...
+        </strong>
+    </li>
+    <li>
+        <strong>
+            Sie können sich auch mit SARS-CoV-2 infizieren, ohne dass die App
+            Sie über die Risiko-Begegnung mit der Person informiert, die Sie
+            infiziert hat:
+        </strong>
+    </li>
+</ul>
```
Comment (SabineLoss) [inline Corona-Warn-App/src/main/assets/terms_de.html:335]: Is the md syntax correct here? The bulleted lists ends with the third bullet, but the following paragraphs belong to the third bullet.

File: Corona-Warn-App/src/main/assets/terms_de.html [hunk 3]
```diff
@@ -354,37 +402,38 @@ <h1>
...
+<p>
     6. AUSLÖSEN EINER WARNUNG
-</h1>
+</p>
 <p>
-    Sie können über die App andere Kontaktpersonen warnen, wenn Sie mit
-    SARS-CoV-2 infiziert wurden.
+    Sie können über die App andere Kontaktpersonen warnen, wenn Sie positiv auf
+    eine Infektion mit SARS-CoV-2 getestet wurden. <strong>Sie dürfen diese Warnung nur auslösen,
+    wenn </strong>
```
Comment (SabineLoss) [inline Corona-Warn-App/src/main/assets/terms_de.html:411]: Can /strong and strong be removed here?

File: Corona-Warn-App/src/main/assets/terms_de.html [hunk 4]
```diff
@@ -605,29 +665,27 @@ <h4>
...
+</p>
 <p>
-    Das RKI behält sich vor, diese Nutzungsbedingungen zu ändern. In diesem
-    Fall werden Sie beim Start der App aufgefordert, den geänderten
-    Nutzungsbedingungen zuzustimmen. Wenn Sie den geänderten
-    Nutzungsbedingungen nicht zustimmen, können Sie die App und die CWA-
-    Dienste nicht mehr nutzen und müssen die App von Ihrem Smartphone löschen.
+    Das RKI behält sich vor, diese Nutzungsbedingungen zu ändern. Wenn Sie den
+    geänderten Nutzungsbedingungen nicht zustimmen, können Sie die App und die
+    CWA- Dienste nicht mehr nutzen und müssen die App von Ihrem Endgerät
```
Comment (SabineLoss) [inline Corona-Warn-App/src/main/assets/terms_de.html:674]:
```suggestion
    CWA-Dienste nicht mehr nutzen und müssen die App von Ihrem Endgerät
```

File: Corona-Warn-App/src/main/assets/terms_de.html [hunk 5]
```diff
@@ -245,61 +284,66 @@ <h1>
...
     </strong>
     Sie besagt lediglich, dass sie während der letzten 14 Tage eine
     Risiko-Begegnung mit einer anderen Person hatten, bei der eine
     SARS-CoV-2-Infektion positiv festgestellt wurde. Hieraus ergibt sich für
     Sie zunächst nur eine Möglichkeit, dass Sie sich ebenfalls infiziert haben
     könnten. Die Einstufung des diesbezüglichen Risikos als "niedrig" oder
-    "erhöht" erfolgt allein auf der Basis der ermittelten Daten und lässt keine
-    Aussage über das tatsächliche Vorliegen einer Infektion zu.
+    "erhöht" erfolgt allein auf der Basis der vom
+    COVID-19-Benachrichtigungssystem und den CWA-Diensten (z. B. Angaben zum
```
Comment (SabineLoss) [inline Corona-Warn-App/src/main/assets/terms_de.html:296]:
```suggestion
    COVID-19-Benachrichtigungssystem und den CWA-Diensten (z.B. Angaben zum
```

File: Corona-Warn-App/src/main/assets/terms_de.html [hunk 6]
```diff
@@ -527,70 +591,66 @@ <h4>
...
+</p>
+<p>
+    Die App ist das alleinige und ausschließliche Eigentum von SAP SE,
+    Dietmar-Hopp-Allee 16, 69190 Walldorf („<strong>SAP</strong>“) oder deren
+    Lizenzgebern, vorbehaltlich der Ihnen nach Ziffer 9 eingeräumten Rechte
+    sowie sonstiger Rechte an der App, die SAP der Bundesrepublik Deutschland
+    vertraglich eingeräumt hat.
+</p>
+<p>
+    <strong><em>Open Source-Lizenzhinweise</em></strong>
```
Comment (SabineLoss) [inline Corona-Warn-App/src/main/assets/terms_de.html:605]:
```suggestion
    <strong><em>Open-Source-Lizenzhinweise</em></strong>
```

File: Corona-Warn-App/src/main/assets/terms_de.html [hunk 7]
```diff
@@ -527,70 +591,66 @@ <h4>
...
 </p>
-<h4>
-    Open Source-Lizenzhinweise
-</h4>
 <p>
     Information über die Verwendung von Drittkomponenten in der App und die
     anwendbaren Lizenzbestimmungen finden Sie in den Open
```
Comment (SabineLoss) [inline Corona-Warn-App/src/main/assets/terms_de.html:609]:
```suggestion
    anwendbaren Lizenzbestimmungen finden Sie in den Open-
```

**Comment by harambasicluka** [comment]
Will mrge this, if agreed we'll update it at the beginning of next week

**Pull request merged**